apps/laboratory/src/components/Wagmi/WagmiSendUSDCTest.tsx

```diff
@@ -1,97 +1,100 @@
 import { Button, Stack, Link, Text, Spacer, Input } from '@chakra-ui/react'
-import { useAccount, useWriteContract } from 'wagmi'
-import { useCallback, useState } from 'react'
-import { optimism, sepolia } from 'wagmi/chains'
+import { useAccount, type Config } from 'wagmi'
+import { useState } from 'react'
+import { arbitrum, base, optimism, sepolia } from 'wagmi/chains'
```

Comment:
We can use `@reown/appkit/networks`, they are literally same but would be nice to show users our own approach

Reply:
Added! Good point